configs/ozone_config/frontend/ozone-frontend-config.json

```diff
@@ -1,8 +1,8 @@
 {
   "@openmrs/esm-patient-registration-app": {
     "sections": [
-      "demographicsSection",
-      "contactSection",
+      "demographics",
+      "contact",
       "additionalInformation"
```

Comment:
I'd leave the names of these sections because they are definite code-wise  and perhaps rename `additionalInformation` to `additonalInformationSection`

Reply:
This was done based on a conversation with @vasharma05 [here ](https://mekomsolutions.slack.com/archives/G421UNF5L/p1698746172767799?thread_ts=1698734503.031209&cid=G421UNF5L)

Reply:
Cc: @kdaud

Reply:
Hi @kdaud !
Code-wise, the section names doesn't have `Section` appended to it. Please have a look [here](https://github.com/openmrs/openmrs-esm-patient-management/blob/44fd35646288e072ad68dac25d6b5e62ce8ea9c7/packages/esm-patient-registration-app/src/config-schema.ts#L79)